### PR TITLE
Added introduction and revised links to the README

### PR DESCRIPTION
**Pull request by GBKS**
Expanded on the introduction sentence to better explain how the role of this repo in the wider community setup.
Tried to make links and descriptions overall more precise and up-to-date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
 
-### Guide
+## Bitcoin Design Guide
 
 - Project Intro — [Doc v2.0.0](https://docs.google.com/document/d/1YiYeRIybGmxmErCOI4Jc8Qajz3JGM1JYVfUtpzyCzSk/edit?usp=sharing)
 - Slack Channel — [#bitcoin-design-guide](https://bitcoindesign.slack.com/archives/C015856BDME)
 - Github Repository — [Guide](https://github.com/BitcoinDesign/Guide)
 - Github Project Board — [Guide/Content](https://github.com/BitcoinDesign/Guide/projects/1)
+
+## Relevant Slack channels
```
Comment (danielnordh) [inline README.md:30]: ## Further Slack channels

**Review by danielnordh — COMMENTED** [review]
Looks good to me, much improved.
Only suggested changing the last headline from 'Relevant' to 'Further' Slack channels, as we already list some other relevant ones further up.

File: README.md [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
-#### How to get involved in the community
+Community activity is fluid, but here are a few important starting points:
 
-1. [Join the community](https://join.slack.com/t/bitcoindesign/shared_invite/zt-gytq2snl-4TEWJOTKrXRCB4YLBoDunA) on Slack
-2. Learn a pr [Project Life Cycle]((Projects.md))
-3. Discuss and shape [processes for the community and open design](https://github.com/BitcoinDesign/Meta/issues)
+## How to get involved in the community
 
-#### How to get involved in projects
+1. [Join the community on Slack](https://join.slack.com/t/bitcoindesign/shared_invite/zt-gytq2snl-4TEWJOTKrXRCB4YLBoDunA) on Slack
```
Comment (jonatack) [inline README.md:12]: remove the redundant "on Slack" after the link

**Comment by johnsBeharry** [comment]
LGMT

File: README.md (hunk 1 above)
Comment (GBKS) [reply README.md]: Thanks. Fixed.

File: README.md (hunk 2 above)
Comment (GBKS) [reply README.md]: Thanks. Fixed.

**Review by johnsBeharry — APPROVED** [review]

**Pull request merged**